.github/scripts/rust/Makefile.lint.toml

```diff
@@ -0,0 +1,3 @@
+[tasks.update-clippy]
+script_runner = "@rust"
+script = { file = "${CARGO_MAKE_CURRENT_TASK_INITIAL_MAKEFILE_DIRECTORY}/lint.rs", absolute_path = true }
```

Comment:
I guess this will always be executed from the toolchain which is used to invoke `cargo make`?